crates/core/src/client/helpers.rs

```diff
@@ -250,17 +249,11 @@ pub fn decode_signature_from_tx_calldata(calldata: &[FieldElement]) -> Result<Si
 ///
 /// TODO: add error case message
 pub fn felt_option_to_u256(element: Option<&FieldElement>) -> Result<U256, EthApiError> {
```

Comment:
I feel like this function should not exist?
As per our newest type conversions,
maybe let's create a standalone issue to delete `felt_option_to_u256` and `felt_to_u256` and refactor with our newtype flow